cirq/google/xmon_device_test.py

```diff
@@ -155,7 +160,7 @@ def test_validate_scheduled_operation_adjacent_exp_11_exp_w():
             cirq.CZ(q1, q2), cirq.Timestamp(), d),
     ])
     d.validate_schedule(s)
-
+    
```

Comment:
Nit: go into pycharm settings and find the option that automatically removes trailing space when you save the file.